test/benchee/benchmark/runner_test.exs

```diff
@@ -131,6 +131,23 @@ defmodule Benchee.Benchmark.RunnerTest do
       assert length(memory_usages) > 0
     end
 
+    @tag :memory_measure
+    test "correctly scales down memory usage of very fast functions" do
```

Comment:
I don't totally understand what this is testing - can you explain?

Reply:
explanation here: https://github.com/PragTob/benchee/pull/196#discussion_r174155669

Reply:
Ahh, that makes sense - thanks for the explanation!